### ISSUE TITLE
Also add aspectJ library, if (integration)test-block in build.gradle is empty

### ISSUE THREAD

**Issue by mawHBT** [jenkinsci/peass-ci-plugin#202]
### What feature do you want to see added?

If you set onlyOneCallRecording, kieker-xxx-aspectj.jar has to be added to jvmArgs in build.gradle. This does not work, if no (integration)test-block is present.

**Comment by mawHBT** [comment]
As first steps, I added a test for checking if aspectJ-library is added to jvmArgs if no test-block is present in [PR #101](https://github.com/DaGeRe/peass/pull/101). This currently fails.

**Comment by mawHBT** [comment]
TestJVMArgsGradle was green, but testing with a HBT-project failed. The appropriate modules had test- and integrationTest-tasks but no appropriate blocks with any configurations. jvmArgs were added, systemProperties inside it, but no aspectJ-line (which should because of onlyOneCallRecording). Unfortunately I was not able to provide a test to reproduce this. But with [PR#105](https://github.com/DaGeRe/peass/pull/105) this now works also. I checked build.gradle of the modules for different combinations of (empty/not present)(IT)test-blocks and jvmArgs set or not.
